pkg/devfile/parser/data/v2/2.2.0/devfileJsonSchema220.go

```diff
@@ -457,6 +457,10 @@ const JsonSchema220 = `{
               }
             ],
             "properties": {
+              "autoBuild": {
+                "description": "Defines if the image should be built during startup Default is false",
```

Comment:
`Default is false` should be a separate sentence.  Can we update it to `Default value is 'false'` so it's consistent with other descriptions? https://github.com/devfile/library/blob/f276a013d254bcdbe90327013fa659d1cbc019f0/pkg/devfile/parser/data/v2/2.2.0/devfileJsonSchema220.go#L200

Reply:
This requires api change, PR created: https://github.com/devfile/api/pull/718
Can you help review and approve that PR?

Reply:
Yes, I'll review it

Reply:
updated the PR with the latest schema which includes the description change